### ISSUE TITLE
Impaired renal tubular reabsorption of sodium 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6864]
**Preferred term label:**
Impaired renal tubular reabsorption of sodium 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of sodium.
PMID: 29237739

**Comment** 
99% of filtered sodium is reabsorbed in the tubule system.

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Associated term**
Renal Fanconi syndrome HP:0001994

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
@RichardKlaus  Is this something that is actually measured (in which case it is a good HPO term) or is it an inference about the pathophysiology (in which case it is not a good HPO term)? We should make the HPO term label reflect the actual measurement as much as possible.

**Comment by RichardKlaus** [comment]
This can actually be measured indirectly (it is calculated from urine and serum sodium, urine and serum creatinine).
If sodium reabsorption is below 99% is pathological.

Von: Peter Robinson ***@***.***>
Gesendet: Freitag, 30. April 2021 13:06
An: obophenotype/human-phenotype-ontology ***@***.***>
Cc: Klaus, Richard Dr.med. ***@***.***>; Mention ***@***.***>
Betreff: Re: [obophenotype/human-phenotype-ontology] Impaired renal tubular reabsorption of sodium (#6864)


@RichardKlaus<https://github.com/RichardKlaus> Is this something that is actually measured (in which case it is a good HPO term) or is it an inference about the pathophysiology (in which case it is not a good HPO term)? We should make the HPO term label reflect the actual measurement as much as possible.

**Comment by pnrobinson** [comment]
So do you mean this:
https://medlineplus.gov/ency/article/003602.htm
Could we add a commentary about this so that users will know what terms to use? I have not been able to find any great references.

**Comment by pnrobinson** [comment]
added
